src/components/Dashboard/Roles.js

```diff
@@ -0,0 +1,105 @@
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import {
+  DashboardLink as DashLink,
+  DashboardLayout as Layout
+} from 'components/Dashboard/';
+import { Table } from 'react-bootstrap';
+import { PagerInfo, withPagination } from 'components/Pagination/Pagination';
+import { actions as rolesActions } from 'reducers/roles';
+import { getAllRoles, getRolesPager } from 'selectors/roles';
+
+export class Roles extends Component {
+  static propTypes = {
+    layoutOptions: PropTypes.object.isRequired,
+    collection: PropTypes.array.isRequired,
+    pager: PropTypes.object.isRequired,
+    pagerNavigation: PropTypes.node.isRequired
+  };
+
+  render() {
+    const { collection, layoutOptions, pager, pagerNavigation } = this.props;
+    // Administrator and User roles aren't editable
+
+    let noEdit = false;
```

Comment:
Would you be able to refactor `noEdit` to a getter so that the assignment logic is not in the render method?

Reply:
Easy fix